### ISSUE TITLE
Error in: curl -SLsf https://dl.get-arkade.dev/ | sudo sh

### ISSUE THREAD

**Issue by christiaanpauw** [openfaas/workshop#216]
Minor documentation error. See official arkade github: 
https://github.com/alexellis/arkade

**Comment by alexellis** [comment]
Yep this has changed. Do you want to send a commit to change it? Just make sure you use the sign off.
